docs/operations/debug.md

```diff
@@ -0,0 +1,59 @@
+# Debug
+This page contains instructions on how to debug Kubevirt.
```

Comment:
For consistency, could you do a search/replace for `s/Kubevirt/KubeVirt/g` ?

Reply:
Done, thanks!